src/app/pages/stacks/stacks-create/stacks-create.component.ts

```diff
@@ -42,37 +37,15 @@ export class StacksCreateComponent implements OnInit {
     };
   }
 
-  clearAlerts() {
-    this.warning = false;
-  }
-
   submit() {
-    this.alreadyExists = false;
-    this.genericError = false;
-    this.submitted = true;
-
     this.stackService
       .deployStack(this.stackModel.stackName, btoa(this.stackModel.stackFile))
       .subscribe(
         (result: StackError) => {
-          this.clearAlerts();
-          this.submitted = false;
-          this.router.navigate([
```

Comment:
We still need to navigate to the Stack List view after successfully deploying a stack, but we can remove `{ createdStack: this.stackModel.stackName },`

Reply:
Does one want to go back directly after deploying a stack?

Consider the situation where you want to deploy 3 stacks. This means that you have to go to the page every time to deploy your new stack?

Or do we assume a person won't deploy more than one stack at a time that often?